geowebcache/core/src/main/java/org/geowebcache/config/GeoWebCacheConfiguration.java

```diff
@@ -227,13 +227,15 @@ public void setProxyUrl(String proxyUrl) {
     }
 
     /**
+     * @see ServerConfiguration#getServiceInformation()
      * @return the serviceInformation
      */
     public ServiceInformation getServiceInformation() {
         return serviceInformation;
     }
 
     /**
+     * @see
```

Comment:
Missing an actual reference here.
Also, getFormatModifiers() and setFormatModifiers() (below) should probably also reference ServerConfiguration